src/function/scalar/string/like.cpp

```diff
@@ -24,6 +34,22 @@ struct NotLikeEscapeOperator {
 	}
 };
 
+struct ILikeOperator {
+	template <class TA, class TB, class TR> static inline TR Operation(TA str, TB pattern) {
+		const std::string& s = tolower(str);
```

Comment:
Please avoid the use of std::string in functions such as this; using std::string adds a lot of overhead with data copying and data duplication. Instead, look at how `caseconvert.cpp` handles this: first figure out the length of the string, then allocate a buffer, then write the resulting string there.